README.md

```diff
@@ -105,7 +105,7 @@ Just do it. This entire Modlist is in English and 99% of all mods you will find
 
 Open the Steam Properties window, navigate to the _Language_ tab and select _English_ from the dropdown menu.
 
-#### Clean Skyrim <Special Edition>
```

Comment:
`<stuff>` is treated as an HTML tag, so the "Special Edition" text never showed up before. I guess we want it?

Reply:
Agree